### ISSUE TITLE
Question: I want to have my H2s not be folded by default

### ISSUE THREAD

**Issue by jdsimcoe** [sdelements/node-slate#1]
Loving `node-slate` but I want my H2s not to collapse and expand by default, only H3s. How would I go about this?

**Comment by sibartlett** [comment]
I think you'd need to modify `source/javascripts/app/_toc.js`, but I am not sure exactly how. I realize this is not very helpful 😞

**Comment by MikeRalphson** [comment]
@jdsimcoe might be worth asking in [Ruby Slate](https://github.com/lord/slate/issues), as this applies to all versions of Slate using tocify? I had a quick look at the [tocify website](http://gregfranko.com/jquery.tocify.js/#OptionsAPI), but I couldn't see any way to keep the H2s in the TOC but all expanded by default - if that's what you meant?

If you only want H1s and H3s in the TOC you can change `source/javascripts/app/_toc.js` to have

``` javascript
  var makeToc = function() {
    global.toc = $("#toc").tocify({
      selectors: 'h1, h3',
```

**Comment by jdsimcoe** [comment]
@MikeRalphson Yeah I want the H2s in there but everything (except expanded except the H3s)

**Comment by jdsimcoe** [comment]
Thanks @sibartlett, I'll look into it.
